src/config.rs

```diff
@@ -1,29 +1,30 @@
 use std::fs;
 use std::io::ErrorKind::NotFound;
 
+use ansi_term::Color;
 use serde::{Deserialize, Serialize};
 
-use crate::memory::MemoryUnit;
+use crate::memory::Memory;
+use crate::shell::Shell;
 
 const BINARY_NAME: &str = env!("CARGO_PKG_NAME");
 
 #[derive(Serialize, Deserialize)]
 #[serde(default)]
```

Comment:
note that this default only applies to **values** inside the struct (you need a separate `#[serde(default)]` for the structs)